docs/docs/api/06-Configuration/askui-ui-control-client.md

```diff
@@ -0,0 +1,80 @@
+---
+custom_edit_url: null
+---
+# askui UI Control Client
+
+## Table of Contents
+
+  - [Properties](#properties)
+    - [uiControllerUrl](#uiControllerUrl)
+    - [inferenceClientUrl](#inferenceClientUrl)
+    - [annotationLevel](#annotationLevel)
+    - [credentials](#credentials)
+  - [Set Log Level](#set-log-level)  
+  - [Example](#example)  
+## Properties
+
+### uiControllerUrl
+
+• **uiControllerUrl**: `string` - Default: `http://localhost:6769`
+
+This provides the url where the askui UI Controller runs. The askui UI Control Client is also connecting
+to this address.
```

Comment:
Seems like we can shorten this to `The url of the askui UI Controller controlling the OS.` What do you think?